src/sider/senoppfolging/KandidatSvar.tsx

```diff
@@ -1,18 +1,53 @@
-import { SenOppfolgingFormResponseDTOV2 } from "@/data/senoppfolging/senOppfolgingTypes";
 import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
-import { BodyShort, Box, Heading, Label } from "@navikt/ds-react";
+import {
+  Alert,
+  BodyShort,
+  Box,
+  Heading,
+  Label,
+  Loader,
+} from "@navikt/ds-react";
 import React from "react";
+import { useSenOppfolgingSvarQuery } from "@/data/senoppfolging/useSenOppfolgingSvarQuery";
+import { SvarResponseDTO } from "@/data/senoppfolging/senOppfolgingTypes";
 
 const texts = {
   heading: "Sykmeldtes svar",
+  formResponse: {
+    pending: "Henter spørsmål og svar...",
+    error:
+      "Noe gikk galt ved henting av spørsmål og svar. Vennligst prøv igjen senere.",
+  },
 };
 
+function KandidatFormResponse() {
+  const { data, isPending, isError } = useSenOppfolgingSvarQuery();
+
+  if (isPending) {
+    return <Loader size="xlarge" title={texts.formResponse.pending} />;
+  }
+  if (isError || !data) {
+    return (
+      <Alert size="small" inline variant="error">
+        {texts.formResponse.error}
+      </Alert>
+    );
+  }
+
+  return data.questionResponses.map((response, index) => (
+    <div key={index}>
+      <Label size="small">{response.questionText}</Label>
+      <BodyShort size="small">{response.answerText}</BodyShort>
+    </div>
+  ));
+}
+
```

Comment:
Har prøvd å finne gode retningslinjer for hva som er bra practice og ikke så bra practive i react, og synes airbnb sin styleguide har mye bra.
De [skriver blant annet at man bare skal ha en react komponent per fil (utenom hvis man har stateless eller pure)](https://arc.net/l/quote/ovzrkcjw). Skal vi prøve å følge dette? 😄 Merker selv at det blir litt mer forvirring om man splitter opp i flere komponenter i samme fil. Her synes jeg `KandidatSvar` er såpass enkel at det bare kunne levd sammen 🤔 Eventuelt hatt en pure komponent for spørsmål/svar par. Hva tenker du? 😄

Reply:
Ja, er stort sett enig i det, men akkurat i dette tilfellet hvor `KandidatSvar` alltid skal rendre heading og tekst mens `KandidatFormResponse` render avhengig av `data, isPending, isError` synes jeg det egentlig blir mer oversiktlig å ha det i en egen komponent.

Reply:
Der `KandidatFormResponse` blir brukt, hadde det også vært et alternativ å gjøre noe slik:
```react
<KandidatFormResponse>
    <KandidatQuestionsAndAnswers />
</KandidatFormResponse>
```
Slik at vi ikke nøster så langt nedover at det blir vanskelig å holder konteksten det blir brukt i.